include/mbedtls/sha256.h

```diff
@@ -232,15 +249,25 @@ int mbedtls_sha256_ret( const unsigned char *input,
 #else
 #define MBEDTLS_DEPRECATED
 #endif
+
 /**
- * \brief          Output = SHA-256( input buffer )
+ * \brief          This function calculates SHA-256 
```

Comment:
Suggestion: Uniformize with `mbedtls_sha256_ret`, saying `This function calculates the SHA-224 or SHA-512 checksum of a buffer.`.

Reply:
changed, with SHA-256 instead of SHA-512 (I assume this was a copy&paste mistake)

Reply:
Yes, thank you!